### ISSUE TITLE
Header icon always to 🤔

### ISSUE THREAD

**Issue by WengerK** [antistatique/windmill#29]
### 💬 Is your feature request related to a problem? Please describe.

It seems the icon is always visible on a non-future weeks

See in the current week the icon is already visible on monday
> <img width="535" alt="Screenshot 2020-06-08 at 08 36 26" src="https://user-images.githubusercontent.com/1841592/83999635-6c2d8380-a963-11ea-9a7f-02d1db99464c.png">

The icon is also visible on past week
> <img width="263" alt="Screenshot 2020-06-08 at 08 36 58" src="https://user-images.githubusercontent.com/1841592/83999647-70f23780-a963-11ea-8aa6-0c415fb22ae5.png">

### 🧞 Describe the solution you'd like

The icon should be binded with the `au` column named "Why?`.
When this column show an icon, display it here.

**Comment by JeremyGfeller** [comment]
Done in the commit aa9a625

**Comment by WengerK** [comment]
mmhh ... I remember discussing with you about not hardcoding the idea of mustSmiley but fetching the icon from the spreadsheet - Why not implementing that  ?

**Comment by JeremyGfeller** [comment]
When the user arrives on the page, all the data is taken from the API, so the smiley (Why?) too. When we add a time, a call to the API is made only to add the value. At the beginning of the project, when we did this action I reloaded all the data in the spreadsheet, but I removed this function to optimize performance. That's why, after adding data, I have no way to resume the spreadsheet smiley.
